src/test/scala/org/kibanaLoadTest/GatlingRunner.scala

```diff
@@ -0,0 +1,21 @@
+package org.kibanaLoadTest
+
+import io.gatling.app.Gatling
+import io.gatling.core.config.GatlingPropertiesBuilder
+
+object GatlingRunner {
+
+  def main(args: Array[String]): Unit = {
+
+    // this is where you specify the class you want to run
+    val simClass =
+      classOf[org.kibanaLoadTest.simulation.branch.DemoJourney].getName
+
```

Comment:
Good point. Actually this is IDEA debug only thing :) Gatling is not an easy thing to debug since it has its own maven lifecycle, but this direct way works perfectly fine.

Let me try pass it as argument in debug window

Reply:
Sweet!